src/dbnode/storage/util.go

```diff
@@ -49,7 +49,7 @@ func timesInRange(startInclusive, endInclusive time.Time, windowSize time.Durati
 		return nil
 	}
 	times := make([]time.Time, 0, ni)
-	for t := endInclusive; !t.Before(startInclusive); t = t.Add(-windowSize) {
```

Comment:
Were there assumptions elsewhere in the code that relies on the time ranges always including the end time? Also fix the comment for this function.